library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
                 **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Alpha-activity-morphology <nowiki> {suggestedTag=Finding-frequency, suggestedTag=Finding-amplitude} [EEG rhythm in the alpha range (8-13 Hz) which is considered part of the background (ongoing) activity but does not fulfill the criteria of the posterior dominant rhythm (alpha rhythm).]</nowiki>
+            *** Alpha-activity <nowiki> {suggestedTag=Finding-frequency, suggestedTag=Finding-amplitude} [Rhythmic activity in the alpha frequency range (8-13 Hz), but not a part of the posterior dominant rhythm. (Source: Beniczky ea 2017, Table 5; Beniczky ea 2013, Appendix S2, S6.)]</nowiki>
                 **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Beta-activity-morphology <nowiki> {suggestedTag=Finding-frequency, suggestedTag=Finding-amplitude} [EEG rhythm between 14 and 40 Hz, which is considered part of the background (ongoing) activity but does not fulfill the criteria of the posterior dominant rhythm. Most characteristically: a rhythm from 14 to 40 Hz recorded over the fronto-central regions of the head during wakefulness. Amplitude of the beta rhythm varies but is mostly below 30 microV. Other beta rhythms are most prominent in other locations or are diffuse.]</nowiki>
+            *** Beta-activity <nowiki> {suggestedTag=Finding-frequency, suggestedTag=Finding-amplitude} [Rhythmic activity in the beta frequency range (14-40 Hz). (Source: Beniczky ea 2017, Table 5; Beniczky ea 2013, Appendix S2, S6.)]</nowiki>
                 **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Gamma-activity-morphology {suggestedTag=Finding-frequency,suggestedTag=Finding-amplitude}
+            *** Gamma-activity <nowiki> {suggestedTag=Finding-frequency,suggestedTag=Finding-amplitude} [Rhythmic activity in the gamma frequency range. (Source: Beniczky ea 2017, Table 5.)]</nowiki>
                 **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
 
-        ** Spike-morphology <nowiki>[A transient, clearly distinguished from background activity, with pointed peak at a conventional paper speed or time scale and duration from 20 to under 70 ms, i.e. 1/50-1/15 s approximately. Main component is generally negative relative to other areas. Amplitude varies.]</nowiki>
+        ** Spike-morphology <nowiki>[A transient, clearly distinguished from background activity, with pointed peak at a conventional paper speed or time scale and duration from 20 to under 70 ms, i.e. 1/50-1/15 s approximately. Main component is generally negative relative to other areas. Amplitude varies. (Source: Beniczky ea 2017, Table 5; Beniczky ea 2013, Appendix S4.)]</nowiki>
```

Comment:
Not sure what "conventional paper speed" means in this.

Reply:
Paper speed is a legacy term from when electrophysiology was recorded on graph paper moving at a certain speed. See for example: https://www.wikidoc.org/index.php/Electrocardiogram section on ecg graph paper